src/helper/remote_config/protocol/client.hpp

```diff
@@ -0,0 +1,36 @@
+// Unless explicitly stated otherwise all files in this repository are
+// dual-licensed under the Apache-2.0 License or BSD-3-Clause License.
+//
+// This product includes software developed at Datadog
+// (https://www.datadoghq.com/). Copyright 2021 Datadog, Inc.
+#pragma once
+
+#include <list>
+#include <string>
+
+#include "client_state.hpp"
+#include "client_tracer.hpp"
+
+namespace dds::remote_config::protocol {
```

Comment:
It's not strictly necessary to add a namespace per directory, for example you might find that there's no value to having `::protocol` (or maybe there is).

Reply:
Ok. That namespace per directory thinking comes from my old php days

Reply:
Done